### ISSUE TITLE
Epoch confusion in 10_neural_nets_with_keras

### ISSUE THREAD

**Issue by ssuhas76** [ageron/handson-ml2#372]
hello @ageron 

I am running my first keras code in Jupyter notebook. The first code in the chapter is for the SequentialAPI(). Though i understand it clearly on a theoretical basis I am having hard time understanding the output of the model.fit. Why is the epoch size differing in the outputs below? 

`history = model.fit(X_train,y_train,epochs=30,
                    validation_data=(X_valid,y_valid))`

Output of my Jupyter notebook is as below.

Epoch 1/30
1719/1719 [==============================] - 4s 2ms/step - loss: 1.0187 - accuracy: 0.6807 - val_loss: 0.5207 - val_accuracy: 0.8234
Epoch 2/30
1719/1719 [==============================] - 3s 2ms/step - loss: 0.5028 - accuracy: 0.8260 - val_loss: 0.4345 - val_accuracy: 0.8538
Epoch 3/30
1719/1719 [==============================] - 3s 2ms/step - loss: 0.4485 - accuracy: 0.8423 - val_loss: 0.5288 - val_accuracy: 0.8002
Epoch 4/30
1719/1719 [==============================] - 3s 2ms/step - loss: 0.4210 - accuracy: 0.8531 - val_loss: 0.3915 - val_accuracy: 0.8650
Epoch 5/30
1719/1719 [==============================] - 3s 1ms/step - loss: 0.4061 - accuracy: 0.8584 - val_loss: 0.3753 - val_accuracy: 0.8684

whereas for the same code the output as per the textbook and the author's code is as below. 

Train on 55000 samples, validate on 5000 samples
Epoch 1/30
55000/55000 [==============================] - 2s 44us/sample - loss: 0.7226 - accuracy: 0.7641 - val_loss: 0.5073 - val_accuracy: 0.8320
Epoch 2/30
55000/55000 [==============================] - 2s 39us/sample - loss: 0.4844 - accuracy: 0.8321 - val_loss: 0.4541 - val_accuracy: 0.8478
Epoch 3/30
55000/55000 [==============================] - 2s 39us/sample - loss: 0.4414 - accuracy: 0.8464 - val_loss: 0.4373 - val_accuracy: 0.8508
Epoch 4/30
55000/55000 [==============================] - 2s 40us/sample - loss: 0.4129 - accuracy: 0.8549 - val_loss: 0.4170 - val_accuracy: 0.8562
Epoch 5/30
55000/55000 [==============================] - 2s 38us/sample - loss: 0.3927 - accuracy: 0.8616 - val_loss: 0.3825 - val_accuracy: 0.8646

Where am i going wrong here?

**Comment by jamesonrader** [comment]
You are running on 55000 batches compared to the books 1719 batches. Maybe you have a different batch size or are using more data samples.
